metrics_utility/management/commands/build_report.py

```diff
@@ -61,43 +87,34 @@ def init_logging(self):
         handler.setLevel(logging.DEBUG)
         handler.setFormatter(logging.Formatter('%(message)s'))
         self.logger.addHandler(handler)
-        self.logger.propagate = False
-
-    def handle(self, *args, **options):
+        self.logger.propagate = True
+
+    def _parse_param(
+        self,
+        param_name,
+        options,
+    ):
+        param = options.get(param_name)
+        if (options.get('month') is not None or param is None) and param_name != 'ephemeral':
+            return None
+        if param:
+            return parse_date_param(param) if parser.parse(param) else parse_number_of_days(param)
```

Comment:
```suggestion
    def _parse_param(
        self,
        param_name,
        options,
    ):
        param = options.get(param_name)
        if (options.get('month') is not None or param is None):
            return None
        return parse_date_param(param)
```

* because `param_name != 'ephemeral'` is always true, we never call it with that param_name now
* if param is None, we return none in the first if already
* and `parse_number_of_days` returns and integer number of days, where since & until expect a datetime, which is handled by `parse_data_param` (`parse_number_of_days` is just for ephemeral)